app/scripts/views/object/perma-link.js

```diff
@@ -0,0 +1,24 @@
+import React, {PropTypes} from 'react'
+import {Link} from 'react-router'
+
+import i18n from '../../utils/i18n'
+import Path from '../../utils/path'
+
+const PermaLink = ({url}) => {
+  if (!url) return <span></span>
```

Comment:
Because otherwise react is unhappy, as this is a functional component, returning `null` or `''` does not work and react throws an error.